.github/workflows/needs-rebase.yml

```diff
@@ -0,0 +1,17 @@
+---
+name: "Pull Request Needs Rebase?"
+on:
+  pull_request_target:
+    types: [opened, synchronize, reopened]
+jobs:
+  needs-rebase:
+    runs-on: ubuntu-latest
+    steps:
+      # eps1lon/actions-label-merge-conflict@v2.0.1
+      - name: Check if PR needs rebase
+        uses: eps1lon/actions-label-merge-conflict@b8bf8341285ec9a4567d4318ba474fee998a6919
+        with:
+          repoToken: "${{ secrets.GITHUB_TOKEN }}"
+          dirtyLabel: "needs-rebase"
+          removeOnDirtyLabel: "ready-to-merge"
```

Comment:
I'm wondering if the `needs-review` label should also be remove before and re-added after the rebase.

Reply:
Mmm, I'm afraid this GH Action doesn't support adding/removing multiple labels... Let's see if we can automate the `needs-review` label in a separate PR with another Action.

Reply:
I see. Thank you :smile: